packages/nc-gui/composables/useSharedFormViewStore.ts

```diff
@@ -37,7 +37,7 @@ const [useProvideSharedFormStore, useSharedFormStore] = useInjectionState((share
   const sharedView = ref<ViewType>()
   const sharedFormView = ref<FormType>()
   const meta = ref<TableType>()
-  const columns = ref<(ColumnType & { required?: boolean; show?: boolean; label?: string })[]>()
+  const columns = ref<(ColumnType & { required?: boolean; show?: boolean; label?: string; enable_scanner?: boolean })[]>()
```

Comment:
use `BoolType` instead of `boolean`

Reply:
Done (also changed the other types to the new wrapper types - that's ok?

Reply:
they are indeed

```
          "label": {
            "$ref": "#/components/schemas/StringOrNull",
            "description": "Form Column Label"
          },
          "required": {
            "$ref": "#/components/schemas/Bool",
            "description": "Is this form column required in submission?"
          },
          "show": {
            "$ref": "#/components/schemas/Bool",
            "description": "Is this column shown in Form?"
          },
          "enable_scanner": {
            "$ref": "#/components/schemas/Bool",
            "description": "Indicates whether the 'Fill by scan' button is visible for this column or not.",
            "example": true
          },
```